### PR TITLE
Added About Dimitri Under Section D

### PR DESCRIPTION
**Pull request by RyanDimitriRamos**
This is just a simple update to the information about Dimitri on the about us page. No CSS changes were made just the information on the header and in the unordered list.

### CODE REVIEW

File: src/main/webapp/aboutus.html [hunk 1]
```diff
@@ -21,7 +21,7 @@
     <title>About Us</title>
     <link rel="stylesheet" href="/css/main.css">
   </head>
-  </head>
+  <Body>
```
Comment (stekolla) [inline src/main/webapp/aboutus.html]: Use a lowercase B in <body> (it doesn't really make a difference, but let's go with all lower case tag names as part of our style guide).

**Review by stekolla — CHANGES_REQUESTED** [review]

File: src/main/webapp/aboutus.html (hunk 1 above)
Comment (RyanDimitriRamos) [reply src/main/webapp/aboutus.html]: Thanks for letting me now. Just to make sure, is this the style guide that you're referring to https://github.com/google/styleguide ?

Comment (stekolla) [reply src/main/webapp/aboutus.html]: I wasn't, but if we want to adopt those guides for our project, that sounds good to me. :)

**Review by stekolla — APPROVED** [review]

File: src/main/webapp/aboutus.html (hunk 1 above)
Comment (RyanDimitriRamos) [reply src/main/webapp/aboutus.html]: I'm not sure if everyone wants to use these, but this is what I found on the codeU documentation. It's a bit hard to understand some of them so, do you have a link to some other style guide?

**Pull request merged**